test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1179,9 +1179,16 @@ class test_ucp_sockaddr_wireup_fail : public test_ucp_sockaddr_wireup {
              * procedure */
             ucp_ep_set_failed(e.ep(), UCP_NULL_LANE, UCS_ERR_ENDPOINT_TIMEOUT);
         } else {
-            for (auto lane = 0; lane < ucp_ep_num_lanes(e.ep()); ++lane) {
-                set_iface_failure(e.ep()->uct_eps[lane]->iface,
-                                  fail_wireup_type);
+            if (fail_wireup_type == FAIL_WIREUP_CONNECT_TO_EP) {
+                /* Go over UCT endpoints created on UCP endpoint and replace
+                 * their uct_ep_connect_to_ep function to make sure that
+                 * 'ucp_wireup_ep_connect_to_ep()' will be failed one, since it
+                 * could return OK without calling inner uct_ep_connect_to_ep
+                 * when UCP endpoint is not reconfigured */
+                for (auto lane = 0; lane < ucp_ep_num_lanes(e.ep()); ++lane) {
```

Comment:
the comment is not clear, do you mean wireup ep?

Reply:
done